bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/device/CommandHandler.java

```diff
@@ -386,13 +386,13 @@ public void handleCommand(InsteonChannelConfiguration conf, Command cmd, Insteon
             try {
                 int button = this.getIntParameter("button", -1);
```

Comment:
Is this variable used anymore then?

Reply:
No, this was leftover from previous usage and I've now removed it from new code